src/cascadia/TerminalApp/CascadiaSettingsSerialization.cpp

```diff
@@ -21,9 +22,10 @@ using namespace ::Microsoft::Console;
 static constexpr std::wstring_view FILENAME { L"profiles.json" };
 static constexpr std::wstring_view SETTINGS_FOLDER_NAME{ L"\\Microsoft\\Windows Terminal\\" };
 
-static constexpr std::wstring_view PROFILES_KEY{ L"profiles" };
-static constexpr std::wstring_view KEYBINDINGS_KEY{ L"keybindings" };
-static constexpr std::wstring_view SCHEMES_KEY{ L"schemes" };
+static constexpr std::string_view PROFILES_KEY{ "profiles" };
+static constexpr std::string_view KEYBINDINGS_KEY{ "keybindings" };
+static constexpr std::string_view GLOBALS_KEY{ "globals" };

```

Comment:
Is globals the right name for this? It's like application settings... but also, i dunno, why _aren't_ they at root level?

Reply:
frankly I just wanted to bundle them up. now that we lose the ordering we had before, the file would contain something like 
```
{
  "global1": val, 
  "global2": val, 
  "profiles": [
     { ... },
     { ... },
  ],
  "global3": val,
  "colorSchemes": [
    { ... }
  ],
  "global4": val,
}
```

with the globals intermixed with things like the profiles, color schemes, etc, which I thought was annoying. I can switch it back if we want.